PopupMenuDemo/MainActivity.cs

```diff
@@ -0,0 +1,36 @@
+namespace PopupMenuDemo;
+
+[Activity(Label = "PopupMenuDemo", MainLauncher = true)]
+public class Activity1 : Activity
+{
+    protected override void OnCreate(Bundle? bundle)
+    {
+        base.OnCreate(bundle);
+
+        SetContentView(Resource.Layout.Main);
+
+        Button showPopupMenu = FindViewById<Button>(Resource.Id.popupButton)!;
+            
+        ArgumentNullException.ThrowIfNull(showPopupMenu);
+
+        showPopupMenu.Click += (s, arg) => {
```

Comment:
Alternatively, if "differently bad", you could just use `showPopupMenu?.Click += …`, and *ignore* `showPopupMenu` being `null`.

*Or*, if `NullReferenceException` *is* deemed useful -- and sometimes it is! -- then "lean into" it by using `showPopupMenu!.Click += …`.